build.xml

```diff
@@ -29,6 +29,12 @@
         </exec>
     </target>
 
+    <target name="create-index" depends="prepare" description="Create index">
```

Comment:
teď na to koukám, phpunit si bere nastavení z `phpunit.xml.dist` a `phpunit-full.xml.dist`, `build.xml` se použije jen v jenkinsi a tam už to zařídí capistrano, takže lokálně to stejně ničemu nepomůže, nebo vy to někdo lokálně testujete `ant`em nebo něčím podobným?